skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/SakBrev.kt

```diff
@@ -171,6 +172,19 @@ fun Route.sakBrev(dto2ApiService: Dto2ApiService, brevredigeringService: Brevred
                 ?.onError { error, _ -> call.respond(HttpStatusCode.InternalServerError, error) }
                 ?: call.respond(HttpStatusCode.NotFound, "Fant ikke PDF")
         }
+
+        route("/{brevId}/p1") {
```

Comment:
Mistenkjer at vi vil ha ein delete også, for viss brukaren ønskjer å fjerne manuelle overstyringar. Kanskje verdt å avklare med Karoline.